app/services/pii/password_encryptor.rb

```diff
@@ -12,6 +12,7 @@ def encrypt(plaintext, user_access_key)
     end
 
     def decrypt(ciphertext, user_access_key)
+      raise EncryptionError unless sane_payload?(ciphertext)
```

Comment:
Should we create a separate `DecryptionError` class? Or add a message here, ex "can't decrypt incorrectly delimited payload" ?